test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -113,6 +108,22 @@ const setUpResponse = (filePath) => {
   // Majority of the mock files can be loaded with just mocking the introspect API and providing a response
   // In some cases we may need to mock more than just introspect API to load the screen (example polling)
   // For those cases add the url and responses to responseMap
+
+  // Needed for identify-with-apple-redirect-sso-extension
+  const verifyUrl = 'http://localhost:3000/idp/idx/authenticators/sso_extension/transactions/123/verify?\
```

Comment:
This this need to be a JWT or can it simply be any opaque value?

Reply:
I am not sure, I just copied the setup from an existing test

Reply:
This mock seems to be bit flaky on bacon. Not sure what the right set up is. I am going to add it to ignore list and have home team investigate later.